airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/typing_deduping/JdbcDestinationHandler.kt

```diff
@@ -340,22 +348,28 @@ abstract class JdbcDestinationHandler<DestinationState>(
         }
     }
 
-    private fun isAirbyteRawIdColumnMatch(existingTable: TableDefinition): Boolean {
-        return existingTable.columns.containsKey(JavaBaseConstants.COLUMN_NAME_AB_RAW_ID) &&
-            toJdbcTypeName(AirbyteProtocolType.STRING) ==
-                existingTable.columns[JavaBaseConstants.COLUMN_NAME_AB_RAW_ID]!!.type
+    protected open fun isAirbyteRawIdColumnMatch(existingTable: TableDefinition): Boolean {
+        return toJdbcTypeName(AirbyteProtocolType.STRING)
+            .equals(
+                existingTable.columns[JavaBaseConstants.COLUMN_NAME_AB_RAW_ID]!!.type,
```

Comment:
I think the kotlin way is existingTable.columns.getValue(JavaBaseConstants.COLUMN_NAME_AB_RAW_ID).type